print/print-apps/crdppf/config.yaml

```diff
@@ -1,4 +1,4 @@
-throwErrorOnExtraParameters: true
+throwErrorOnExtraParameters: false
```

Comment:
Is that OK for you ? (or have you changed that for testing purposes?)

Reply:
Yes, I prefer it to be set to false by default and activate it, in case of errors 👍